cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringHandler.java

```diff
@@ -101,10 +112,24 @@ public void getTether(HttpRequest request, HttpResponder responder, @PathParam("
   @DELETE
   @Path("/tethering/connections/{peer}")
   public void deleteTether(HttpRequest request, HttpResponder responder, @PathParam("peer") String peer)
-    throws PeerNotFoundException, IOException {
+    throws NotFoundException, IOException, BadRequestException {
+    boolean tetheringServerEnabled = cConf.getBoolean(Constants.Tethering.TETHERING_SERVER_ENABLED);
```

Comment:
Does it make sense to have a default value of `false` here in case we are unable to fetch this value?

Reply:
The default value is set in cdap-default.xml
https://github.com/cdapio/cdap/blob/develop/cdap-common/src/main/resources/cdap-default.xml#L5180